rebar.config

```diff
@@ -15,7 +15,7 @@
     {rebar3_hank, "~> 1.3.0"}
 ]}.
 
-{deps, [{sasl_auth, "2.0.0-rc1"}]}.
+{deps, [{sasl_auth, "2.0.1"}]}.
```

Comment:
I think you meant this to be an rc of 2.0.0, but tbh, I'm good on sasl_auth as a non-rc at this point. Thoughts?

Reply:
https://github.com/kafka4beam/brod_gssapi/issues/17#issuecomment-1221237267

I forgot it was still on RC, after we had to fix a bug, I pushed a 2.0.1, should have been 2.0.0 or 2.0.0-rc1

Reply:
Yeah, I guess what I was trying to say, since it's already published, we might just want to go with it since we can't delete it, best we can do is retire it, but that just be a last resort. Thoughts?